Extensibility/TemplatePack/ProjectTemplates/CSharp/Wpf/PrismAutofacApp/PrismAutofacApp.vstemplate

```diff
@@ -0,0 +1,53 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<VSTemplate Version="3.0.0" Type="Project" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
+  <TemplateData>
+    <Name>Prism Autofac App (WPF)</Name>
+    <Description>A project template for building WPF applications with Prism and Autofac.</Description>
+    <Icon>PrismAutofacApp.ico</Icon>
+    <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>2.0</RequiredFrameworkVersion>
+    <TemplateID>4064d4a5-f80e-432a-9f4b-7917e3bb49a7</TemplateID>
```

Comment:
All template ID's must be unique.  Please use a different GUID for each TemplateID in every .vstemplate file in all the other templates you have submitted.